server/api/api/endpoints/nuclio.py

```diff
@@ -477,6 +478,13 @@ def _deploy_nuclio_runtime(
 
         if serving_to_monitor:
             if not mlrun.mlconf.is_ce_mode():
+                if fn.spec.image == "mlrun/mlrun" and semver.Version.parse(
+                    client_version
+                ) < semver.Version.parse("1.6.3"):
+                    raise mlrun.errors.MLRunBadRequestError(
+                        "Serving function can't be deploy with `mlrun/mlrun` image and "
+                        "client version lower when 1.6.3 while set tracking is on."
+                    )
```

Comment:
TMI
perhaps "Serving function with set-tracking feature is not supported for mlrun clients <= 1.6.2" would be enough?

Reply:
the image is also relevant no?